packages/builder/src/pages/builder/app/[application]/_components/BuilderSidePanel.svelte

```diff
@@ -52,15 +55,32 @@
   }
 
   const filterInvites = async query => {
+    if (!prodAppId) {
+      return
+    }
+
     appInvites = await getInvites()
-    if (!query || query == "") {
-      filteredInvites = appInvites
+
+    //On Focus behaviour
+    if (!filterByAppAccess && (!query || query == "")) {
```

Comment:
NAB but just a quick FYI that `(!query || query == "")` is redundant - `!query` by itself will already account for empty strings.